packages/listbox/src/ListboxMixin.js

```diff
@@ -381,6 +385,21 @@ const ListboxMixinImplementation = superclass =>
       /* eslint-enable no-param-reassign */
     }
 
+    resetInteractionState() {
+      super.resetInteractionState();
+      this.submitted = false;
+    }
+
+    reset() {
+      this.formElements.forEach(formElement => {
```

Comment:
It should be possible to do: `this.modelValue = this._initialModelValue` (since it's SSOT and will automatically sync checked state under the hood)

Reply:
Yes indeed, although it required a small fix under that hood :)